### PR TITLE
Enable application property page to open in designer

### PR DESCRIPTION
**Pull request by davkean**
- Moved designer related code -> Designer.vb code
- Enabled property page to open in the designer, and let the designer fix up values in the resx.

### CODE REVIEW

**Comment by davkean** [comment]
tag @dotnet/roslyn-analysis

File: src/Microsoft.VisualStudio.Editors/PropPages/ApplicationPropPage.resx [hunk 1]
```diff
@@ -597,9 +597,6 @@
   <data name="ManifestExplanationLabel.Size" type="System.Drawing.Size, System.Drawing">
     <value>467, 28</value>
   </data>
-  <data name="ManifestExplanationLabel.AutoSize" type="System.Boolean, mscorlib">
-    <value>True</value>
```
Comment (davkean) [inline src/Microsoft.VisualStudio.Editors/PropPages/ApplicationPropPage.resx]: This is a text box, not a label, hence why this disappeared

File: src/Microsoft.VisualStudio.Editors/PropPages/ApplicationPropPage.resx [hunk 2]
```diff
@@ -819,17 +816,17 @@
...
   </data>
   <data name="Win32ResourceRadioButton.Margin" type="System.Windows.Forms.Padding, System.Windows.Forms">
     <value>0, 4, 0, 0</value>
   </data>
   <data name="Win32ResourceRadioButton.Size" type="System.Drawing.Size, System.Drawing">
-    <value>93, 17</value>
-  </data>
-  <data name="Win32ResourceRadioButton.AutoSize" type="System.Boolean, mscorlib">
-    <value>True</value>
+    <value>91, 17</value>
```
Comment (davkean) [inline src/Microsoft.VisualStudio.Editors/PropPages/ApplicationPropPage.resx]: This autosized in the designer, dropping 2 "designer" px

**Comment by davkean** [comment]
Added VB's WinForm's version to this review as well.

File: src/Microsoft.VisualStudio.Editors/PropPages/ApplicationPropPage.Designer.vb [hunk 3]
```diff
@@ -0,0 +1,272 @@
+﻿
+Namespace Microsoft.VisualStudio.Editors.PropertyPages
+
+    <Global.Microsoft.VisualBasic.CompilerServices.DesignerGenerated()>
+    Partial Class ApplicationPropPage
+
+        Private components As System.ComponentModel.IContainer
```
Comment (basoundr) [inline src/Microsoft.VisualStudio.Editors/PropPages/ApplicationPropPage.Designer.vb]: `_components` to maintain consistent naming?

**Comment by basoundr** [comment]
:+1:

**Pull request merged**